site/content/community/_index.html

```diff
@@ -0,0 +1,111 @@
+---
+title: "Community"
+layout: section
+---
+
+<div class="hero subpage-hero">
+  <div class="wrapper">
+    <h1>Community</h1>
+  </div>
+</div>
+<div class="wrapper subpage">
+  <h2>Do you want to help build Kubeapps?</h2>
+  <div class="grid three">
+    <div class="col">
+      <div class="icon">
+        <img alt="Github logo" src="/img/github-image.svg" />
+      </div>
+      <div class="content">
+        <h3>
+          <a target="_blank" rel="noopener" href="https://github.com/vmware-tanzu/kubeapps/"
```

Comment:
The templating {{ params.xxxx }} don't work here, it would have to be a `partial` not a `content`, but the template was this way. Let's change in the future maybe.